src/index.js

```diff
@@ -19,3 +19,10 @@ ReactDOM.render(
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
+const btn = document.querySelector('#toggle-navbar');
+const menu = document.querySelector('#mobile-menu');
+
+btn.addEventListener('click', () => {
+  menu.classList.toggle('hidden');
+});
```

Comment:
What's the purpose of those lines here ?

Reply:
This icon was responsible for the hamburger button, however I change this code to another code using `useRef` hook.